docs/wiki/framework/crew-served-weapons-framework.md

```diff
@@ -136,7 +138,7 @@ class CfgVehicles {
     class prefix_hmg: StaticMGWeapon {
         class ACE_CSW {
             enabled = 1; // Enables ACE CSW for this weapon              
-            proxyWeapon = "prefix_hmg_weapon_proxy"; // The proxy weapon created above
+            proxyWeapon = "prefix_hmg_weapon_proxy"; // The proxy weapon created above. This can also be a function name that returns a proxy weapon
```

Comment:
```suggestion
            proxyWeapon = "prefix_hmg_weapon_proxy"; // The proxy weapon created above. This can also be a function name that returns a proxy weapon - passed [_vehicle, _turret, _currentWeapon, _needed, _emptyWeapon]
```